src/neo4j/cypher-queries/character.js

```diff
@@ -67,7 +67,10 @@ const getShowQuery = () => `
 					writingCredits: sourceMaterialWritingCredits
 				}
 			END
-		) | CASE entity.model WHEN 'MATERIAL'
+		) AS entities
+
+	WITH character, materialRel, material, writingCreditName,
+		[entity IN entities | CASE entity.model WHEN 'MATERIAL'
```

Comment:
This message would appear for this section's former state:
`This feature is deprecated and will be removed in future versions.`
`Aggregation column contains implicit grouping expressions. Aggregation expressions with implicit grouping keys are deprecated and will be removed in a future version. For example, in 'RETURN n.a, n.a + n.b + count(*)' the aggregation expression 'n.a + n.b + count(*)' includes the implicit grouping key 'n.b', and this expression is now deprecated. It may be possible to rewrite the query by extracting these grouping/aggregation expressions into a preceding WITH clause.`